applications/sintering/include/pf-applications/sintering/postprocessors.h

```diff
@@ -1381,6 +1381,136 @@ namespace Sintering
       data_out.write_vtu_in_parallel(output, dof_handler.get_communicator());
     }
 
+    template <int dim, typename VectorType>
```

Comment:
Could you add a short comment what this function does and what pores are? How are these different from `grains`, `particles`, ...?

Reply:
Yep, done.